lib/routes/routeBackbeat.js

```diff
@@ -89,17 +89,23 @@ function _checkMultipleBackendRequest(request, log) {
         log.error(errMessage);
         return errors.BadRequest.customizeDescription(errMessage);
     }
-    if ((operation === 'puttagging' || operation === 'deletetagging') &&
-        headers['x-scal-source-bucket'] === undefined) {
-        errMessage = 'bad request: missing x-scal-source-bucket header';
-        log.error(errMessage);
-        return errors.BadRequest.customizeDescription(errMessage);
-    }
-    if ((operation === 'puttagging' || operation === 'deletetagging') &&
-        headers['x-scal-source-version-id'] === undefined) {
-        errMessage = 'bad request: missing x-scal-version-id header';
-        log.error(errMessage);
-        return errors.BadRequest.customizeDescription(errMessage);
+    if (operation === 'puttagging' || operation === 'deletetagging') {
+        if (headers['x-scal-data-store-version-id'] === undefined) {
```

Comment:
is it always `''` coming from backbeat and not `undefined` if we do not have the replicated version id stored in the kafka entry?

Reply:
It will be `''`.

Reply:
Replication info in the Kafka entry when the tagging is put after replication:

```
{\"status\":\"PENDING\",\"content\":[\"METADATA\",\"PUT_TAGGING\"],\"destination\":\"arn:aws:s3:::destination-bucket\",\"storageClass\":\"awsbackend\",\"role\":\"arn:aws:iam::887847504243:role/RoleForReplication-1510950202478\",\"storageType\":\"aws_s3\",\"dataStoreVersionId\":\"0hPAGSGpo7o03eQpKfvivoVHJHKlNkBJ\"}
```

and when tagging is put before replication:

```
{\"status\":\"PENDING\",\"content\":[\"METADATA\",\"PUT_TAGGING\"],\"destination\":\"arn:aws:s3:::destination-bucket\",\"storageClass\":\"awsbackend\",\"role\":\"arn:aws:iam::887847504243:role/RoleForReplication-1510951189163\",\"storageType\":\"aws_s3\",\"dataStoreVersionId\":\"\"}
```